ts/util/table-builder.ts

```diff
@@ -169,10 +224,73 @@ export class TableBuilder<ParentType, ChildType> {
 
   private addFilterControls(container: HTMLElement) {
     const filterContainer = document.createElement("div");
-    filterContainer.classList.add("flex-auto");
-    container.appendChild(filterContainer);
+    filterContainer.classList.add("flex-auto", "space-x-2", "items-center");
```

Comment:
Would a `space-y-2` or something improve display when wrapping happens? Currently, the +filter button can overlap with the accepted filter elements

Reply:
![image](https://user-images.githubusercontent.com/58961173/179791594-14201e4e-2423-48c6-8df4-8aa189f6460e.png)
space-y-2 messes up the top margins unfortunately

Reply:
Changing the accepted filter elements to inline-block actually fixes the spacing too